WordPress/Classes/ViewRelated/Post/PostListFilter.swift

```diff
@@ -101,6 +111,16 @@ import Foundation
         let predicate = NSPredicate(format: "status IN %@", statuses.strings)
         let title = NSLocalizedString("Trashed", comment: "Title of the trashed filter. This filter shows posts that have been moved to the trash bin.")
 
-        return PostListFilter(title: title, filterType: filterType, predicate: predicate, statuses: statuses)
+        let filter = PostListFilter(title: title, filterType: filterType, predicate: predicate, statuses: statuses)
+        filter.accessibilityIdentifier = "trashed"
+
+        return filter
+    }
+}
+
+extension PostListFilter: FilterTabBarItem {
+
```

Comment:
In lieu of manually setting the accessibility identifier for each `PostListFilter` instance, I wonder if you could add something like the following to this extension?

```
var accessibilityIdentifier: String {
    return filterTitle.lowercased() // filterTitle.localizedLowercase might have less utility for automation
}
```

Reply:
If you like this idea, then _I think_ it could even be added as a protocol extension to `FilterTabBarItem`. :)

Reply:
It's a good idea, but the `filterTitle` may be a localized string, while we need the `accessibilityIdentifier` to be a constant string so that the test suite can consistently identify it.

That's why I didn't add a protocol extension for that functionality – I was worried it'd cause folks to omit the manual one. I was thinking about conditional conformance (so if it's on an `enum: String` or `enum: Int` it could just determine a logical value automatically) but I was worried it might be too magical. WDYT – would it be worth doing to prevent having to manually provide a value?

Reply:
If we intend to run automation tests with varying locale, then you're right - my suggestion isn't so valid.

Your protocol extension idea is valid. It seems to me that offering the default behavior (e.g., return `"\(self)"` for `Int` types & `rawValue` for `String` types) would encourage consistency for future automation tests.

Ultimately I'll leave it to your discretion. I appreciate that you considered it!

Reply:
Added in https://github.com/wordpress-mobile/WordPress-iOS/pull/11172/commits/bec2ce3a21746e32797ca113eb46c8492582f1a1 – thanks so much for the review and discussion! :)